packages/@lwc/features/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@lwc/features",
+    "version": "1.0.2",
+    "description": "LWC Features",
+    "main": "dist/commonjs/flags.js",
+    "module": "dist/esm/flags.js",
+    "license": "MIT",
+    "scripts": {
+        "build": "tsc && tsc --project tsconfig.commonjs.json",
+        "clean": "rm -rf dist/"
+    },
+    "files": [
+        "dist/"
+    ],
+    "dependencies": {
+        "@babel/core": "~7.5.5",
```

Comment:
Why not use the @babel/core version from the root package.json?
@lwc/template-compiler also has @babel/template as a dependency, it can be lifted up to the root package.json

Reply:
I think it's ok to lift up devDeps to the root, but we should be explicit with dependencies.

Reply:
yes, this should definitely be dev deps.

Reply:
Was able to eliminate the dependency altogether by moving the rollup plugin wrapper into engine.